app/models/viewmodels/notificationspage/ResponseErrorExplanationSuffixProvider.scala

```diff
@@ -0,0 +1,31 @@
+/*
+ * Copyright 2019 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package models.viewmodels.notificationspage
+
+import config.AppConfig
+import javax.inject.Inject
+
+class ResponseErrorExplanationSuffixProvider @Inject()(appConfig: AppConfig) {
+
+  private val DefaultSuffix = ".Exports"
+
+  lazy val suffix: String = appConfig.responseErrorExplanationMode match {
+    case "CDS"     => ".CDS"
+    case "Exports" => ".Exports"
```

Comment:
This case is unnecessary. Default value cover it.

Reply:
Why not put the 'dot' in the config value and just use that (i.e. no need for this provider class).  Unless you want to prevent spreading the AppConfig around.

Alternatively, this provider can just return s".${appConfig.responseErrorExplanationMode}"

Reply:
I know it is not necessarty, but this is more about documenting what do we expect. In this case it is just a coincidence that the default value is the same.